crates/criticaltrust/src/v2/snapshots/criticaltrust__v2__manifests__tests__release_package_json_serialize.snap

```diff
@@ -0,0 +1,35 @@
+---
+source: crates/criticaltrust/src/v2/manifests.rs
```

Comment:
Are instas something we actually use?

```
criticalup  main ?via 🦀 v1.88.0 
❯ cargo insta review
done: no snapshots to review

```

Reply:
The current snapshot matches the snapshot output from the `source` test. In this case `cargo insta review` outputs `done: no snapshots to review`